src/LibPQ.jl

```diff
@@ -13,6 +13,8 @@ using DocStringExtensions
 using Decimals
 using FileWatching
 using Tables
+using Infinity
+using Infinity.Utils
```

Comment:
What is needed from this?

Reply:
I thought I had to import it as it defines some definitions for `isinf` but I guess not.